cmd/nodecmd/stop.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	awsAPI "github.com/ava-labs/avalanche-cli/pkg/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+
+	"github.com/spf13/cobra"
+)
+
+func newStopCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "stop [clusterName]",
+		Short: "(ALPHA Warning) Stop all nodes in a cluster",
+		Long: `(ALPHA Warning) This command is currently in experimental mode.
```

Comment:
You should add a note that stopping nodes may be insufficient to stop future charges and you may still be incur further costs